src/components/Nav.tsx

```diff
@@ -72,21 +72,21 @@ const Nav = (props: { network: string; isPro?: boolean }) => {
                         {t("history")}
                     </A>
 
-                    <Show when={config.docsUrl}>
+                    <Show when={config.discordUrl}>
```

Comment:
```suggestion
                    <Show when={config.supportUrl}>
```
Just noticed that this is saying `discordUrl`